DevSetup/app-Ntools.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "1.2.0",
+  "NbuildAppList": [
+    {
+      "Name": "Ntools",
+      "Version": "1.4.7",
+      "AppFileName": "$(InstallPath)\\nb.exe",
+      "WebDownloadFile": "https://github.com/naz-hage/ntools/releases/download/$(Version)/$(Version).zip",
+      "DownloadedFile": "$(Version).zip",
+      "InstallCommand": "powershell.exe",
+      "InstallArgs": "-Command Expand-Archive -Path $(Version).zip -DestinationPath \u0027$(InstallPath)\u0027 -Force",
+      "InstallPath": "$(ProgramFiles)\\Nbuild",
+      "UninstallCommand": "powershell.exe",
+      "UninstallArgs": "-Command Remove-Item -Path \u0027$(InstallPath)\u0027 -Recurse -Force"
+    }
+  ]
+}
```

Comment:
@ahmdhjj It would be a good idea to add more tools like VS 2022.  Think about what development tools do developers need to install so that `gettting-started` project can build and test.

Reply:
@naz-hage would it make sense to add these tools to `apps.json` instead of `apps-Ntools.json`?

Reply:
apps.json

Reply:
Added more apps to apps.json